tip-wlan/charts/nginx-ingress-controller/templates/controller-configmap.yaml

```diff
@@ -7,4 +7,4 @@ metadata:
     {{- include "common.labels" . | nindent 4 }}  
 data:
     external-status-address: {{ .Values.controller.config.externalStatusAddress }}
-    client-max-body-size: {{ .Values.controller.config.clientMaxBodySize }}
\ No newline at end of file
+    client-max-body-size: {{ .Values.controller.config.clientMaxBodySize }}
```

Comment:
Why is this file changed?

Reply:
mm I think editor appended newline .. not important, will revert it